tests/callbacks/test_Matchers.cpp

```diff
@@ -225,7 +225,7 @@ TEST(EventBeginEndIdMatcher, traits)
 //! @test Check the behavior of @ref Kokkos::utils::callbacks::EventBeginEndIdMatcher.
 TEST(EventBeginEndIdMatcher, operator_parentheses)
 {
-    EventBeginEndIdMatcher<EventNameMatcher, BeginParallelForEvent> matcher {.matcher = EventNameMatcher("example-pfor")};
+    EventBeginEndIdMatcher<EventNameMatcher, BeginParallelForEvent> matcher {.matcher = {"example-pfor"}};
```

Comment:
Let's extract these changes to a separate PR :wink:

Reply:
Indeed! It's:
- https://github.com/uliegecsm/kokkos-utils/pull/80